ajax/libs/onecolor/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "onecolor",
+  "filename": "one-color-all.js",
+  "description": "Javascript color object with implicit color space conversions. Supports RGB, HSV, HSL and CMYK with alpha channel.",
+  "license": "BSD-2-clause",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/One-com/one-color.git"
+  },
+  "version": "3.0.1",
+  "keywords": [
+    "color",
+    "colour",
+    "conversion",
+    "rgb",
+    "hsv",
+    "hsl",
+    "cmyk"
+  ],
+  "npmName": "onecolor",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "one-color*"
+      ]
+    }
+  ],
+  "authors": [
+    "Peter Müller",
+    "Andreas Lind"
```

Comment:
@x09326 Any reason you remove the email info of the authors?
https://github.com/One-com/one-color/blob/master/package.json#L18
https://github.com/One-com/one-color/blob/master/package.json#L22